crates/astria-composer/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astria-composer"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Did we have a breaking change here?

Answer: yeah, we had breaking changes in the protobufs, composer uses the protobufs in its public interfaces, hence this needs to bump the protobufs.